pymeasure/display/widgets/__init__.py

```diff
@@ -0,0 +1,12 @@
+from .browser_widget import BrowserWidget
```

Comment:
Could you also add the copyright text over here

Reply:
Yup pushed that up